common/changes/@microsoft.azure/openapi-validator-rulesets/tejaswis-XmsPageableForGetListCalls_2023-09-21-23-34.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft.azure/openapi-validator-rulesets",
+      "comment": "Added XmsPageableForGetListCalls (RPC-Get-V1-13) rule to ensure that all GET LIST APIs contain the ‘x-ms-pageable’ property",
```

Comment:
>Added [](http://example.com/codeflow?start=18&length=5)

Should be Add.
Thats the mandate it seems

Reply:
Not a mandate but a best practice to use simple present tense.